extensions/amp-brightcove/0.1/validator-amp-brightcove.protoascii

```diff
@@ -36,6 +36,13 @@ tags: {  # <amp-brightcove>
     name: "data-account"
     mandatory: true
   }
+  # <amp-bind>
+  attrs: { "[data-account]" }
```

Comment:
That's actually fine since amp-bind doesn't run (evaluate expressions and apply them) on initial page load. I.e. pages that use amp-bind should be valid and render even with all bindings (`[attribute]`) stripped.

This is another difference we'll have to make if we go down the "bind+types" route for A4A.